slt/src/app/pages/authenticated/user/account/account.component.ts

```diff
@@ -41,4 +46,28 @@ export class AccountComponent {
 					});
 		}
 	}
+
+	public deleteAccount() {
+		this.authService.deleteAccount(this.password).subscribe(
+			() => {
+				localStorage.clear();
+				this.router.navigate(['/']);
+			},
+			err => {
+				if (err.status === 401) {
+					this.errorMessage = 'Password is incorrect';
+				} else {
+					// TODO
```

Comment:
een hamburger/toast/popup tonen dat niet gelukt is? of gaat de error interceptor hier een stokje voor steken?

Reply:
daar lijkt het me idd goed voor. maar bij loginscherm juist niet :/ errors horen soms op de pagina en soms een toast. generiek wordt lastig. misschien een errortoaster ?

Reply:
Story aangemaakt voor generieke oplossing op alle pagina's/voor alle service calls. Staat bovenaan.